### ISSUE TITLE
ZenHub workflow documentation

### ISSUE THREAD

**Issue by MaxWofford** [hackclub/meta#577]
We'll be using ZenHub to manage our GitHub issues. 

This issue tracks creating a guide for managing Hack Club's GitHub issues using ZenHub.

**Comment by kyleemile** [comment]
Zen board workflow: 

You’ll notice that there are 7 categories. 

**Unsorted:** 
This is where new task are placed. No items should remain in unsorted for longer than a day 

**Backlog:** 
This contains work that are of low priority. Typically these are things that won’t get done within the next 2 weeks. 

**Todo:** 
These hold items that are queued, and are waiting to be worked on. 

**In Progress:** 
These are items that a person is actively working on. Each team member should only have one item in progress at a time. 

**Blocking:** 
This is for work that cannot be completed until someone else completes a contingent work stream. 

**Review:** 
After a task has been completed, specifically after a pull request has been submitted, it should be moved to review. All work completed should be reviewed by another team member. 

**Closed:**
This is where items that have been completed are placed. 

How the board reads: 

1. We start at the far right at the Review column. Checking if there’s any work that needs to be reviewed by you. 
2. From there you move to Blocking, to make sure that you’re not blocking anyone else from completing their work. 
3. Next up is In Progress, here’s where you should find what you’re currently working on. If there’s any updates or change in the work, please update the board to reflect it. 
4. We then move to Todo, here’s you’ll find all the work you have queued. If you currently have nothing In Progress, move the Todo item with the highest priority over to In Progress and start working on it!
5. Next is Backlog, check that your work here doesn’t need to be moved into To do.
6. Last up is unsorted—review this column and make sure that the work gets properly sorted into either Backlog or Todo. 

Here is the ideal flow of a work item: 

item starts in unsorted > moved to Todo or Backlog > when someone begins working on the item, it is to be moved to the top of In Progress > if item cannot be completed because another piece of work must be completed first, move it to Blocking, if not skip this step > after the item is completed, and pull request has been submitted, it is to be moved to Review.

**Comment by SyncyMcSyncface** [comment]
https://app.asana.com/0/171333429831189/171333429831189, where does this workflow need to be posted?

[--- Commented from Asana.com
#commenter Kyle Emile
---[aa]

**Comment by SyncyMcSyncface** [comment]
It should be documented in the meta repo

[--- Commented from Asana.com
#commenter Max
---[aa]

**Comment by SyncyMcSyncface** [comment]
https://app.asana.com/0/171333429831189/171333429831189 I'm quite sure how to do that....would you mind jumpoing on a call sometime and helping me figure that out?

[--- Commented from Asana.com
#commenter Kyle Emile
---[aa]

**Comment by MaxWofford** [comment]
Sure! How's 1pm PST today?

**Comment by MaxWofford** [comment]
This was closed by #649
